content/en/docs/Installation/_index.md

```diff
@@ -78,6 +78,34 @@ If you'd rather deploy the manifest directly, simply apply the latest release fi
 ```sh
 kubectl create -f https://raw.githubusercontent.com/kyverno/kyverno/main/definitions/release/install.yaml
 ```
+## Verify Kyverno image authenticity using cosign
+We sign the official container images that are built on each commit to the main branch. These are signed using cosign. To verify the container image, download our public key into a file named cosign.pub and then:
+
+To verify, you must have `cosign` installed.
+```sh
```

Comment:
For linting purposes of Markdown, could you please insert new lines before and after code blocks?

Reply:
Resolved @chipzoller